meson.build

```diff
@@ -4357,6 +4365,7 @@ foreach tuple : [
         ['link-systemctl-shared', get_option('link-systemctl-shared')],
         ['link-networkd-shared',  get_option('link-networkd-shared')],
         ['link-timesyncd-shared', get_option('link-timesyncd-shared')],
+        ['link-journalctl-shared',get_option('link-journalctl-shared')],
```

Comment:
Please add a white space after `,`, and shift all other entries.

Reply:
entries shifted by one space now